hack/release

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+: ${RELEASE_OUT=release}
+: ${PFXPASSWORD=}
+
+workdir=$(pwd)
+dockerpfx=$(mktemp -t dockercredhelper-pfx.XXXXXXXXXX)
+
+function clean {
+  rm -f $dockerpfx
+}
+
+trap clean EXIT
+
+cd bin/
+for d in */ ; do
+  os=$(echo ${d::-1} | awk -F'_' '{ print $1 }')
+  arch=$(echo ${d::-1} | awk -F'_' '{ print $2 }')
+  echo "Create release for $os/$arch"
+  if [ "$os" = "linux" ]; then
+    (
+      cd $d
+      set -x
+      tar cvfz "${workdir}/${RELEASE_OUT}/docker-credential-pass-$arch.tar.gz" docker-credential-pass
+      tar cvfz "${workdir}/${RELEASE_OUT}/docker-credential-secretservice-$arch.tar.gz" docker-credential-secretservice
+    )
+  elif [ "$os" = "darwin" ]; then
+    (
+      cd $d
+      set -x
+      #SIGNINGHASH=$(security find-identity -v -p codesigning | grep "Developer ID Application: Docker Inc" | cut -d ' ' -f 4)
+      #xcrun -log codesign -s $(SIGNINGHASH) --force --verbose docker-credential-osxkeychain
+      #xcrun codesign --verify --deep --strict --verbose=2 --display docker-credential-osxkeychain
```

Comment:
We discussed about that with @tonistiigi to sign our buildx binaries. I'm not sure if we should add secrets on the repo to be able to codesign (can be painful) or if we should create our own self-hosted MacOS GitHub Runner with the identity already available (cc @chris-crone).